diff -u --recursive sendmail-8.18.1-vanilla/cf/cf/Makefile sendmail-8.18.1/cf/cf/Makefile
--- sendmail-8.18.1-vanilla/cf/cf/Makefile	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/cf/cf/Makefile	2025-09-17 14:43:14.849079128 -0500
@@ -80,10 +80,10 @@
 install-cf:  install-sendmail-cf install-submit-cf
 
 install-sendmail-cf: $(CF).cf
-	$(INSTALL) -c -o $(CFOWN) -g $(CFGRP) -m $(CFMODE) $(CF).cf ${DESTDIR}$(MAILDIR)/sendmail.cf
+	$(INSTALL) -c -m $(CFMODE) $(CF).cf ${DESTDIR}$(MAILDIR)/sendmail.cf
 
 install-submit-cf: $(SUBMIT).cf
-	$(INSTALL) -c -o $(CFOWN) -g $(CFGRP) -m $(CFMODE) $(SUBMIT).cf ${DESTDIR}$(MAILDIR)/submit.cf
+	$(INSTALL) -c -m $(CFMODE) $(SUBMIT).cf ${DESTDIR}$(MAILDIR)/submit.cf
 
 depend:
 
diff -u --recursive sendmail-8.18.1-vanilla/devtools/M4/UNIX/executable.m4 sendmail-8.18.1/devtools/M4/UNIX/executable.m4
--- sendmail-8.18.1-vanilla/devtools/M4/UNIX/executable.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/devtools/M4/UNIX/executable.m4	2025-09-17 14:43:14.849335693 -0500
@@ -31,7 +31,7 @@
 ifdef(`bldNO_INSTALL', ,
 `install-`'bldCURRENT_PRODUCT: bldCURRENT_PRODUCT ifdef(`bldTARGET_INST_DEP', `bldTARGET_INST_DEP')
 	ifdef(`confMKDIR', `if [ ! -d ${DESTDIR}${bldINSTALL_DIR`'BINDIR} ]; then confMKDIR -p ${DESTDIR}${bldINSTALL_DIR`'BINDIR}; else :; fi ')
-	${INSTALL} -c -o ${bldBIN_TYPE`'BINOWN} -g ${bldBIN_TYPE`'BINGRP} -m ${bldBIN_TYPE`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${bldINSTALL_DIR`'BINDIR}
+	${INSTALL} -c -m ${bldBIN_TYPE`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${bldINSTALL_DIR`'BINDIR}
 ifdef(`bldTARGET_LINKS', `bldMAKE_TARGET_LINKS(${bldINSTALL_DIR`'BINDIR}/bldCURRENT_PRODUCT, ${bldCURRENT_PRODUCT`'TARGET_LINKS})')')
 
 strip-`'bldCURRENT_PRODUCT: bldCURRENT_PRODUCT
diff -u --recursive sendmail-8.18.1-vanilla/devtools/M4/UNIX/library.m4 sendmail-8.18.1/devtools/M4/UNIX/library.m4
--- sendmail-8.18.1-vanilla/devtools/M4/UNIX/library.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/devtools/M4/UNIX/library.m4	2025-09-17 14:43:14.849462183 -0500
@@ -28,7 +28,7 @@
 
 install-`'bldCURRENT_PRODUCT: bldCURRENT_PRODUCT.a
 ifdef(`bldINSTALLABLE', `	ifdef(`confMKDIR', `if [ ! -d ${DESTDIR}${bldINSTALL_DIR`'LIBDIR} ]; then confMKDIR -p ${DESTDIR}${bldINSTALL_DIR`'LIBDIR}; else :; fi ')
-	${INSTALL} -c -o ${LIBOWN} -g ${LIBGRP} -m ${LIBMODE} bldCURRENT_PRODUCT.a ${DESTDIR}${LIBDIR}')
+	${INSTALL} -c -m ${LIBMODE} bldCURRENT_PRODUCT.a ${DESTDIR}${LIBDIR}')
 
 bldCURRENT_PRODUCT-clean:
 	rm -f ${OBJS} bldCURRENT_PRODUCT.a ${MANPAGES}
diff -u --recursive sendmail-8.18.1-vanilla/devtools/M4/UNIX/manpage.m4 sendmail-8.18.1/devtools/M4/UNIX/manpage.m4
--- sendmail-8.18.1-vanilla/devtools/M4/UNIX/manpage.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/devtools/M4/UNIX/manpage.m4	2025-09-17 14:43:14.849590725 -0500
@@ -69,10 +69,10 @@
 define(`bldMAN_INSTALL_CMD',
 `ifdef(`confDONT_INSTALL_CATMAN', `dnl',
 `	ifdef(`confMKDIR', `if [ ! -d ${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)`SRC} ]; then confMKDIR -p ${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)`SRC}; else :; fi ')
-	${INSTALL} -c -o ${MANOWN} -g ${MANGRP} -m ${MANMODE} bldGET_MAN_BASE_NAME($1).`${MAN'bldGET_MAN_SOURCE_NUM($1)`SRC}' `${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)}/bldGET_MAN_BASE_NAME($1)`.${MAN'bldGET_MAN_SOURCE_NUM($1)`EXT}'')
+	${INSTALL} -c -m ${MANMODE} bldGET_MAN_BASE_NAME($1).`${MAN'bldGET_MAN_SOURCE_NUM($1)`SRC}' `${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)}/bldGET_MAN_BASE_NAME($1)`.${MAN'bldGET_MAN_SOURCE_NUM($1)`EXT}'')
 ifdef(`confINSTALL_RAWMAN',
 `	ifdef(`confMKDIR', `if [ ! -d ${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)`MAN} ]; then confMKDIR -p ${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)`MAN}; else :; fi ')
-	${INSTALL} -c -o ${MANOWN} -g ${MANGRP} -m ${MANMODE} bldGET_MAN_BASE_NAME($1).bldGET_MAN_SOURCE_NUM($1) `${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)`MAN}'/bldGET_MAN_BASE_NAME($1)`.${MAN'bldGET_MAN_SOURCE_NUM($1)`EXT}'', `dnl')'
+	${INSTALL} -c -m ${MANMODE} bldGET_MAN_BASE_NAME($1).bldGET_MAN_SOURCE_NUM($1) `${DESTDIR}${MAN'bldGET_MAN_SOURCE_NUM($1)`MAN}'/bldGET_MAN_BASE_NAME($1)`.${MAN'bldGET_MAN_SOURCE_NUM($1)`EXT}'', `dnl')'
 )dnl
 bldFOREACH(`bldMAN_INSTALL_CMD(', `bldMAN_PAGES')
 ifdef(`confNO_MAN_INSTALL', `divert(0)', `dnl')
diff -u --recursive sendmail-8.18.1-vanilla/libmilter/Makefile.m4 sendmail-8.18.1/libmilter/Makefile.m4
--- sendmail-8.18.1-vanilla/libmilter/Makefile.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/libmilter/Makefile.m4	2025-09-17 14:43:14.849742398 -0500
@@ -33,8 +33,8 @@
 MFDEF=	${SRCDIR}/inc`'lude/libmilter/mfdef.h
 install-mfapi: ${MFAPI}
 	if [ ! -d ${DESTDIR}${INCLUDEDIR}/libmilter ]; then mkdir -p ${DESTDIR}${INCLUDEDIR}/libmilter; else :; fi
-	${INSTALL} -c -o ${INCOWN} -g ${INCGRP} -m ${INCMODE} ${MFAPI} ${DESTDIR}${INCLUDEDIR}/libmilter/mfapi.h
-	${INSTALL} -c -o ${INCOWN} -g ${INCGRP} -m ${INCMODE} ${MFDEF} ${DESTDIR}${INCLUDEDIR}/libmilter/mfdef.h
+	${INSTALL} -c -m ${INCMODE} ${MFAPI} ${DESTDIR}${INCLUDEDIR}/libmilter/mfapi.h
+	${INSTALL} -c -m ${INCMODE} ${MFDEF} ${DESTDIR}${INCLUDEDIR}/libmilter/mfdef.h
 divert(0)
 
 bldFINISH
diff -u --recursive sendmail-8.18.1-vanilla/mail.local/Makefile.m4 sendmail-8.18.1/mail.local/Makefile.m4
--- sendmail-8.18.1-vanilla/mail.local/Makefile.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/mail.local/Makefile.m4	2025-09-17 14:43:14.849862805 -0500
@@ -30,7 +30,7 @@
 force-install: install-mail.local ifdef(`confNO_MAN_BUILD',, `install-docs')
 
 install-mail.local: mail.local
-	${INSTALL} -c -o ${UBINOWN} -g ${UBINGRP} -m ${UBINMODE} mail.local ${DESTDIR}${EBINDIR}
+	${INSTALL} -c -m ${UBINMODE} mail.local ${DESTDIR}${EBINDIR}
 divert
 
 bldFINISH
diff -u --recursive sendmail-8.18.1-vanilla/rmail/Makefile.m4 sendmail-8.18.1/rmail/Makefile.m4
--- sendmail-8.18.1-vanilla/rmail/Makefile.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/rmail/Makefile.m4	2025-09-17 14:43:14.849974510 -0500
@@ -31,7 +31,7 @@
 force-install: install-rmail ifdef(`confNO_MAN_BUILD',, `install-docs')
 
 install-rmail: rmail
-	${INSTALL} -c -o ${UBINOWN} -g ${UBINGRP} -m ${UBINMODE} rmail ${DESTDIR}${UBINDIR}
+	${INSTALL} -c -m ${UBINMODE} rmail ${DESTDIR}${UBINDIR}
 divert
 
 bldFINISH
diff -u --recursive sendmail-8.18.1-vanilla/sendmail/Makefile.m4 sendmail-8.18.1/sendmail/Makefile.m4
--- sendmail-8.18.1-vanilla/sendmail/Makefile.m4	2024-01-31 00:38:32.000000000 -0600
+++ sendmail-8.18.1/sendmail/Makefile.m4	2025-09-17 14:43:44.020825814 -0500
@@ -55,8 +55,6 @@
 	@echo "You must have set up a new user ${MSPQOWN} and a new group ${GBINGRP}"
 	@echo "as explained in sendmail/SECURITY."
 	mkdir -p ${DESTDIR}${MSPQ}
-	chown ${MSPQOWN} ${DESTDIR}${MSPQ}
-	chgrp ${GBINGRP} ${DESTDIR}${MSPQ}
 	chmod 0770 ${DESTDIR}${MSPQ}
 
 divert(0)
@@ -68,7 +66,7 @@
 divert(bldTARGETS_SECTION)
 
 install-set-user-id: bldCURRENT_PRODUCT ifdef(`confNO_HELPFILE_INSTALL',, `install-hf') ifdef(`confNO_STATISTICS_INSTALL',, `install-st') ifdef(`confNO_MAN_BUILD',, `install-docs')
-	${INSTALL} -c -o ${S`'BINOWN} -g ${S`'BINGRP} -m ${S`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${M`'BINDIR}
+	${INSTALL} -c -m ${S`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${M`'BINDIR}
 	for i in ${sendmailTARGET_LINKS}; do \
 		rm -f $$i; \
 		${LN} ${LNOPTS} ${M`'BINDIR}/sendmail $$i; \
@@ -76,7 +74,7 @@
 
 define(`confMTA_LINKS', `${DESTDIR}${UBINDIR}/newaliases ${DESTDIR}${UBINDIR}/mailq ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
 install-sm-mta: bldCURRENT_PRODUCT
-	${INSTALL} -c -o ${M`'BINOWN} -g ${M`'BINGRP} -m ${M`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${M`'BINDIR}/sm-mta
+	${INSTALL} -c -m ${M`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${M`'BINDIR}/sm-mta
 	for i in confMTA_LINKS; do \
 		rm -f $$i; \
 		${LN} ${LNOPTS} ${M`'BINDIR}/sm-mta $$i; \
@@ -84,14 +82,14 @@
 
 install-hf:
 	if [ ! -d ${DESTDIR}${HFDIR} ]; then mkdir -p ${DESTDIR}${HFDIR}; else :; fi
-	${INSTALL} -c -o ${UBINOWN} -g ${UBINGRP} -m 444 helpfile ${DESTDIR}${HFFILE}
+	${INSTALL} -c -m 444 helpfile ${DESTDIR}${HFFILE}
 
 install-st: statistics
 	if [ ! -d ${DESTDIR}${STDIR} ]; then mkdir -p ${DESTDIR}${STDIR}; else :; fi
-	${INSTALL} -c -o ${SBINOWN} -g ${UBINGRP} -m ifdef(`confSTMODE', `confSTMODE', `0600') statistics ${DESTDIR}${STPATH}
+	${INSTALL} -c -m ifdef(`confSTMODE', `confSTMODE', `0600') statistics ${DESTDIR}${STPATH}
 
 install-submit-st: statistics ${DESTDIR}${MSPQ}
-	${INSTALL} -c -o ${MSPQOWN} -g ${GBINGRP} -m ifdef(`confSTMODE', `confSTMODE', `0600') statistics ${DESTDIR}${MSPQ}/${MSPSTFILE}
+	${INSTALL} -c -m ifdef(`confSTMODE', `confSTMODE', `0600') statistics ${DESTDIR}${MSPQ}/${MSPSTFILE}
 
 divert(0)
 bldPRODUCT_END
